pkg/services/libraryelements/accesscontrol.go

```diff
@@ -0,0 +1,21 @@
+package libraryelements
+
+import (
+	ac "github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+const (
+	ScopeLibraryPanelsRoot   = "library-panels"
```

Comment:
Note: We've opted for separating other resource names that contain two words with a dot. Eg, "alert.notifications:..." instead of "alert-notifications:...". So it'd be good to follow that pattern for consistency.

Reply:
Nice, thanks for pointing this  out